src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -550,23 +550,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             End If
 
             Dim Here = 1
-            While CanGet(Here)
-                ch = Peek(Here)
-                If IsWhitespace(ch) Then
-                    Here += 1
-                Else
-                    Exit While
+            Dim atNewLine As Boolean
+            Dim FoundCommentAfterUnderscore As Boolean = False
+            ' Line continuation is valid at the end of the line, or at the end of the file, or followed by a trailing comment.
+            ' Eg.  LineContinuation ( EndOfLine | EndOfFile | LineContinuationComment)
+            If CanGet(Here + 1) AndAlso IsWhitespace(Peek(Here)) AndAlso IsSingleQuote(Peek(Here + 1)) Then
```

Comment:
does that mean that only: ```_ ' comment```  is allowed, and not ```_'comment```?

Reply:
i.e. you must have exactly one whitespace between?  What if someone uses multiple whitespace?

Reply:
Yes exactly 1 white space, that was the spec that was proposed and agreed on in VBLang and exactly what I implemented.

Reply:
this just feels strange.  is there any other place in teh language with such a restriction?  i mean, i can write this today in VB:  

![image](https://user-images.githubusercontent.com/4564579/42783557-3cf0a162-8901-11e8-8f4b-29a078eb08ea.png)

But if move to line continuation, i can't write that, and i would have to have:

![image](https://user-images.githubusercontent.com/4564579/42783670-873ca982-8901-11e8-910a-299102c9d4f1.png)

That honestly doesn't make sense to me.  We should allow any amount of whitespace since this is trivia after all..

Reply:
@CyrusNajmabadi I don't disagree, whose decision is this? If this change is agreed to how do the error messages get changed (including Multilanguage).

Reply:
> @CyrusNajmabadi I don't disagree, whose decision is this?

I don't see anywhere where any actual decisions were made :)   So i would just implement something sane and sensible, and write it up for @gafter  to champion.  I'm 99% sure he would be ok with ```underscore can be followed by any number of spaces, then a comment``` as the allowed rule.

Reply:
> If this change is agreed to how do the error messages get changed (including Multilanguage).

You would just change the error yourself as part of the PR.  It's unclear to me what actual errors you would need though (expect for an error that trailing comments after line continuation is in vb16 and above only).